web/src/utilities/utiltyFunctions.ts

```diff
@@ -17,3 +17,35 @@ export function postGisToPoint(location: string): Point | undefined {
   }
   return { x: xCoord, y: yCoord };
 }
+
+export function radiusForZoomLevel(currentZoom: number, minZoom: number, maxZoom: number): number {
+  const inRangeRadius = 10;
+  let diff = 0;
+  if (currentZoom >= minZoom && currentZoom <= maxZoom) {
+    return inRangeRadius;
+  }
+  if (currentZoom < minZoom) {
+    diff = minZoom - currentZoom;
+    if (diff > 2) {
+      return 0;
+    }
+    return inRangeRadius - (5 * diff);
```

Comment:
```suggestion
    // While zooming out, ramp down radius from full size to zero.
    return Math.max(inRangeRadius - 5 * (minZoom - currentZoom), 0.0);
```

This might seem harder to read, but I think it's actually better. It's a more succinct description of what's going on, involving fewer `if`s and intermediate computed values, and the comment helps to guide the reader to exactly what's going on.

Reply:
And actually, an even better approach might be to define an `interpolate()` function somewhere (or use one from the standard library) and use it here directly.